cogs5e/funcs/scripting.py

```diff
@@ -330,6 +330,7 @@ def __init__(self, combatant: Combatant, hidestats=True):
             self.initmod = None
             self.temphp = None
             self.resists = None
+        self.attacks = self._combatant.attacks
```

Comment:
Should attacks only be visible for non-hidden combatants?

Reply:
I was unsure, but I guess not